### ISSUE TITLE
TypeError: ExtensionUtils.initTranslations is not a function Error installing Extension

### ISSUE THREAD

**Issue by terroreek** [gicmo/gamemode-extension#14]
I get the following error with the gnome shell extension.  Running gnome 3.30 on debian unstable with game move version 1.5

```
TypeError: ExtensionUtils.initTranslations is not a function

Stack trace:
  init@/home/terroreek/.local/share/gnome-shell/extensions/gamemode@christian.kellner.me/prefs.js:107:5
  _getExtensionPrefsModule@resource:///org/gnome/shell/extensionPrefs/main.js:68:9
  wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
  _selectExtension@resource:///org/gnome/shell/extensionPrefs/main.js:82:31
  wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
  _onCommandLine@resource:///org/gnome/shell/extensionPrefs/main.js:235:17
  wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
  main@resource:///org/gnome/shell/extensionPrefs/main.js:389:5
  @<main>:1:43
```

**Comment by highvoltage** [comment]
I have the same problem, I tried with a local package and from the extention that's published on the gnoem shell extension site with the same results.

**Comment by terroreek** [comment]
ok, this was resolved for me when gnome was upgraded to 3.34.
